packages/reporter/src/utils/test-file-path-resolver.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'node:path';
```

Comment:
Don't import the entire module as namespace
```suggestion
import { join, relative } from 'node:path';
```